cmd/kubeadm/app/preflight/checks.go

```diff
@@ -803,10 +803,9 @@ func getEtcdVersionResponse(client *http.Client, url string, target interface{})
 				loopCount--
 				return false, err
 			}
-			//lint:ignore SA5011 If err != nil we are already returning.
```

Comment:
glad to see this can get dropped... curious if that's because the new static check is more accurate or for some other reason

Reply:
The staticcheck [docs](https://staticcheck.io/docs/checks#SA5011) say that SA5011 is `A pointer is being dereferenced unconditionally, while also being checked against nil in another place.` So in this particular case, I believe that removing the check for `r != nil` at line 809 makes it so that r is _not_ "also being checked against nil in another place" so SA5011 no longer cares about it.